### PR TITLE
Cleanup native voting system code #1071 Use lifeLongDelegatesProtocol…

### PR DESCRIPTION
**Pull request by lzxm160**
work on #1071 
5、Use lifeLongDelegatesProtocol by default to feed delegate info to make blockchain be able to run by default.

### CODE REVIEW

File: blockchain/blockchain.go [hunk 1]
```diff
@@ -1351,6 +1351,15 @@ func (bc *blockchain) createPollGenesisStates(ctx context.Context, ws factory.Wo
 			ws,
 		)
 	}
+	pollProtocol := poll.NewLifeLongDelegatesProtocol(bc.config.Genesis.Delegates)
```
Comment (zjshen14) [inline blockchain/blockchain.go]: This code is not necessary

File: blockchain/blockchain_test.go [hunk 2]
```diff
@@ -833,8 +833,9 @@ func TestBlockchainInitialCandidate(t *testing.T) {
 	defer func() {
 		require.NoError(bc.Stop(context.Background()))
 	}()
-	// TODO: we will fix this test case by testing using lifeLongDelegatesProtocol to initialize the candidates
-	// candidate, err := sf.CandidatesByHeight(0)
+	candidate, err := sf.CandidatesByHeight(1)
```
Comment (zjshen14) [inline blockchain/blockchain_test.go]: Check this: https://github.com/iotexproject/iotex-core/blob/master/server/itx/server.go#L283-L319

Like the one you fix in minicluster, you need to adjust the genesis config to enable lifeLongDelegatesProtocol

File: blockchain/blockchain_test.go [hunk 3]
```diff
@@ -16,6 +16,7 @@ import (
 	"sync"
 	"testing"
 
+	"github.com/iotexproject/iotex-core/action/protocol/poll"
```
Comment (zjshen14) [inline blockchain/blockchain_test.go]: fix the group

File: blockchain/genesis/genesis.go [hunk 4]
```diff
@@ -172,7 +172,7 @@ type (
 // New constructs a genesis config. It loads the default values, and could be overwritten by values defined in the yaml
 // config files
 func New() (Genesis, error) {
-	def := defaultConfig()
+	def := Default
```
Comment (zjshen14) [inline blockchain/genesis/genesis.go]: No, we should still call defaultConfig()

File: blockchain/blockchain_test.go (hunk 3 above)
Comment (lzxm160) [reply blockchain/blockchain_test.go]: sure

File: blockchain/genesis/genesis.go (hunk 4 above)
Comment (lzxm160) [reply blockchain/genesis/genesis.go]: defaultConfig() does not append delegates,this cause "make run" server error,but Default appended delegates in initTestDefaultConfig()

**Comment by lzxm160** [comment]
move to #1139

**Pull request closed without merging**